### DISCUSSION TITLE
固定見出しは何のためにあるのか

### DISCUSSION THREAD

**Discussion by querykuma** [increments/qiita-discussions#247] [要望・アイデア・不具合]
画面幅の大きな環境では固定見出しが表示されます。
見出しが多い記事で、後ろの見出しを閲覧すると、固定見出しに表示されませんでした。:sob:
例えば、[この記事](https://qiita.com/oioigohan/items/f47f772e936e71f31168)です。
18を閲覧するときに、18以外を閲覧するように固定見出しが誘導しています。:sweat_drops:
固定見出しは何のためにあるのかが問われると思います。
見出しの固定表示をやめるか、固定見出しもスクロールするように変更すれば解決すると思います。:star2:

**Comment by tanaka-asami415** [comment]
@querykuma 
お問い合わせありがとうございます🙏

> 固定見出しもスクロールするように変更すれば

固定見出し→目次のことと解釈いたしました。

目次はスクロールバーでスクロールはできるようになっているのですが、
目次が記事に連動して動かないということでしょうか？
![image](https://user-images.githubusercontent.com/103916185/177133278-ad883c2c-77da-4ff2-aa26-c6fd0279bc76.png)

異なる場合は、どのような現象が起きているのか、画面録画等で教えていただけると幸いです🙇
